src/NuGet/Microsoft.Orleans.Client.nuspec

```diff
@@ -18,9 +18,9 @@
     <copyright>Copyright Microsoft 2015</copyright>
     <tags>Orleans Cloud-Computing Actor-Model Actors Distributed-Systems C# .NET</tags>
     <dependencies>
-      <dependency id="Microsoft.Orleans.Core" version="1.0.3" />
-      <dependency id="Microsoft.Orleans.OrleansAzureUtils" version="1.0.3" />
-      <dependency id="Microsoft.Orleans.OrleansProviders" version="1.0.3" />
+      <dependency id="Microsoft.Orleans.Core" version="$version$" />
```

Comment:
I'll write this a bit more open, if someone else is also wondering...

It looks like this means the client is required always to have the same version of 
* Microsoft.Orleans.Core
* Microsoft.Orleans.OrleansAzureUtils
* Microsoft.Orleans.OrleansProviders

I.e. it's dependencies. This would mean, I gather, those libraries need to updated in sync too. Would this always be the case? Taken from the library names alone, it looks like they could be released individually with differing version numbers (esp. with relation to sematic versioning). Though the reality may be currently they are always released in sync and the having the same version number looks more clear for an outsider.

How to maintain separate packages?